### PR TITLE
Created iosxr-ansible tutorial

### PR DESCRIPTION
**Pull request by Maikor**
This pull request has been automatically generated by prose.io.

### CODE REVIEW

File: _tutorials/2016-05-10-xr-ansible-md.md [hunk 1]
```diff
@@ -0,0 +1,204 @@
+---
+permalink: "/tutorials/iosxr-ansible"
+author: Mike Korshunov
+excerpt: "IOSXR-Ansible tutorial"
+published: true
+title: "XR-Ansible.md"
+---
+# Introduction
+In this tutorial we are going to cover XRv64 configuration with Ansible. We will setup and try simplest configuration with Unix machine connected to XRv64.
```
Comment (akshshar) [inline _tutorials/2016-05-10-xr-ansible-md.md]: XRv64 will not be used as the public name. Use IOS-XRv instead.

File: _tutorials/2016-05-10-xr-ansible-md.md [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+
+## Prerequisites
+- Computer with with 4-5GB free RAM;
+- Vagrant;
+- Ansible;
+- [XRv64 image](http://engci-maven-master.cisco.com/artifactory/appdevci-snapshot/);
```
Comment (akshshar) [inline _tutorials/2016-05-10-xr-ansible-md.md]: Same here. Replace XRv64 throughout the document with IOS-XRv

File: _tutorials/2016-05-10-xr-ansible-md.md [hunk 3]
```diff
@@ -0,0 +1,204 @@
...
+```
+mkorshun@MKORSHUN-2JPYH MINGW64 ~/Documents/workCisco/tutorial
+$ vagrant port xr
+The forwarded ports for the machine are listed below. Please note that
+these values may differ from values configured in the Vagrantfile if the
+provider supports automatic port collision detection and resolution.
+ 22 (guest) = 2223 (host)
+ 57722 (guest) = 2200 (host)
+mkorshun@MKORSHUN-2JPYH MINGW64 ~/Documents/workCisco/tutorial
+$ ssh -p 2223 vagrant@localhost
```
Comment (akshshar) [inline _tutorials/2016-05-10-xr-ansible-md.md]: Mention why you chose port 2223. It maps to port 22 of IOS-XRv, used by XR SSH.

File: _tutorials/2016-05-10-xr-ansible-md.md [hunk 4]
```diff
@@ -0,0 +1,204 @@
...
+vagrant@localhost's password:
+RP/0/RP0/CPU0:ios#
+```
+
+Now let's  configure IP address at XR box. Issue the commant at XR CLI.
```
Comment (akshshar) [inline _tutorials/2016-05-10-xr-ansible-md.md]: *command

File: _tutorials/2016-05-10-xr-ansible-md.md [hunk 5]
```diff
@@ -0,0 +1,204 @@
...
+```
+run vi /root/.ssh/authorized_keys
+
+run sed -i.bak -e '/^PermitRootLogin/s/no/yes/' /etc/ssh/sshd_config_operns
+
+run service sshd_operns restart
+
+run chkconfig --add sshd_operns
+```
+
```
Comment (akshshar) [inline _tutorials/2016-05-10-xr-ansible-md.md]: You cannot login to XR as root, violates security.

1)  User vagrant already exists (that is why vagrant ssh works) and sshd_operns has already been added to chkconfig.

2)  User vagrant has sudo privileges already and does NOT require passwords to execute sudo commands.

3) Use the user vagrant for further tasks, copy the ubuntu public key into .ssh/authorized_keys folder under /home/vagrant

File: _tutorials/2016-05-10-xr-ansible.md [hunk 6]
```diff
@@ -0,0 +1,204 @@
+---
+permalink: "/tutorials/iosxr-ansible"
+author: Mike Korshunov
+excerpt: "IOSXR-Ansible tutorial"
+published: true
+title: "XR-Ansible.md"
```
Comment (akshshar) [inline _tutorials/2016-05-10-xr-ansible.md:6]: Change the title metadata field to "XR-Ansible" , removed md here as well.

**Comment by Maikor** [comment]
I already have one file, it's duplicate, closing PR. 
Comments implemented in another PR. Thanks!

**Pull request closed without merging**